index.html

```diff
@@ -0,0 +1,4741 @@
+<html lang="en">
```

Comment:
Is this extra big file needed in the repo? It seems something autogenerated

Reply:
noooope 😱 removed, thanks